misc/windows-deploy/user-data.ps1

```diff
@@ -0,0 +1,52 @@
+<powershell>
+## The string 'windows' shoud be replaced with 
+## template variable for cluster name.
+
+# Set agent env variables for the Machine context (durrable)
+[Environment]::SetEnvironmentVariable("ECS_CLUSTER", "windows", "Machine")
+[Environment]::SetEnvironmentVariable("ECS_CRED_PROXY_ENABLED", "False", "Machine")
+$agentVersion = '1.14.0-1.windows.1'
+$agentZipHash = '7748b1d3c73d5211e150db746cdec5bf'
+$agentZipUri = "https://s3-us-west-2.amazonaws.com/windows-agent/ecs-agent-windows-$agentVersion.zip"
```

Comment:
We need to change the uri completely. I just need to know where the final uri is.